metarParser-parsers/src/main/java/io/github/mivek/parser/AbstractTAFTrendParser.java

```diff
@@ -36,6 +36,7 @@ void updateTrend(final int index, final String[] parts, final TafTrend trend) {
     for (int i = index; i < parts.length; i++) {
       if (RMK.equals(parts[i])) {
         parseRMK(trend, parts, i);
+        break;
```

Comment:
Should there also be a `break;` after the RMK parsing of the TAF base (not trend) part? For example:

```
TAF CYTL 121940Z 1220/1308 RMK FCST BASED ON AUTO OBS. FCST BASED ON OBS BY OTHER SRCS. WIND SENSOR INOP. NXT FCST BY 130200Z
    TEMPO 1303/1308 2SM -SN
```